documentation/troubleshooting.md

```diff
@@ -0,0 +1,84 @@
+# Troubleshooting common problems
+The following subsections outline the solutions to some common problems that may occur when first setting up a Keptn project to use the dynatrace-service.
+
+## Evaluation succeeds but no evaluation was performed
+
+Evaluation succeeds but the shipyard-controller reports:
+> `no evaluation performed by lighthouse because no SLI-provider configured for project dashboard-config-test`
+
+![No evaluation performed by lighthouse](images/no-evaluation-performed-by-lighthouse.png)
+
+Likely cause:
+- An evaluation was triggered but Keptn is not configured to use Dynatrace as the monitoring provider
+
+Suggested solution:
+- Run `keptn configure monitoring dynatrace --project <project-name>` to configure Dynatrace as the monitoring provider
+
+## Configure monitoring fails
+
+Keptn CLI reports:
+> `Configure monitoring failed. dynatrace-service: cannot handle event: could not get configuration: could not find resource: 'dynatrace/dynatrace.conf.yaml' of project 'dashboard-config-test'`
+
+![Configure monitoring failed](images/configure-monitoring-failed.png)
+
+Likely cause:
+- The CLI command `keptn configure monitoring dynatrace --project <project-name>` was run before adding a `dynatrace/dynatrace.conf.yaml` to the Keptn project
+
+Suggested solution:
+- Create a `dynatrace/dynatrace.conf.yaml` file on the project level. See [Configuring the dynatrace-service with `dynatrace/dynatrace.conf.yaml`](dynatrace-conf-yaml-file.md).
+- Following this, re-run `keptn configure monitoring dynatrace --project <project-name>`
+
+## Evaluation fails: Could not retrieve any SLI results
+
+Evaluation fails but the shipyard-controller reports:
+> `could not retrieve any SLI results`
+
+![Could not retrieve any SLI results](images/could-not-retrieve-any-sli-results.png)
+
+Likely cause:
+- SLIs and SLOs should be defined using YAML files but no SLOs have been defined i.e. no `slo.yaml` file can be found
+
+Suggested solution:
+- If SLIs and SLOs should be defined using YAML files, define SLOs by creating a `slo.yaml` file, or
+- If SLIs and SLOs should be sourced from a dashboard, add a `dashboard` entry to the `dynatrace/dynatrace.conf.yaml` configuration file
```

Comment:
maybe we could add a x-link here to the section that outlines this?

Reply:
Good idea, adding these in #727